src/fsearch_database_entry.c

```diff
@@ -6,6 +6,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#ifdef __MACH__
+int strverscmp (const char *, const char *);
+#endif
```

Comment:
Don't we need to have a `strverscmp.h` instead, which exports `strverscmp`?

Reply:
I guess so, I'm not sure it's needed for one declaration.

Reply:
I think for consistency sake it's better to use a `strverscmp.h` header instead of the forward declaration. After that everything looks fine for merging.

Reply:
Ok, will do.